### PR TITLE
DOC: Add NLG docs to the guide

### PR DESCRIPTION
**Pull request by jaidevd**
ToDo:
- [x] Fix broken links
- [x] Add NLG link in the guide's README.md
- [x] Make the entire NLG component embeddable
- [x] Merge NLG's HTMLs into one HTML

### CODE REVIEW

File: gramex/apps/nlg/gramex.yaml [hunk 1]
```diff
@@ -4,6 +4,23 @@ import:
     YAMLURL: $YAMLURL/ui/
 
 url:
+  nlg-login-$*:
```
Comment (sanand0) [inline gramex/apps/nlg/gramex.yaml]: Use one of the existing GoogleAuths in `../auth/`

File: gramex/apps/nlg/gramex.yaml [hunk 2]
```diff
@@ -70,9 +88,9 @@ url:
     kwargs:
       function: gramex.apps.nlg.webapp.edit_narrative
       auth:
-        login_url: /$YAMLURL/../../login/
+        login_url: /$YAMLURL/login/
```
Comment (sanand0) [inline gramex/apps/nlg/gramex.yaml]: Point this to the same Google login URL we use above

**Comment by jaidevd** [comment]
@sanand0 This should be ready now, pending the pptgen test fix.

**Comment by jaidevd** [comment]
ToDo:
- [ ] Ask users to login only when required.
- [ ] Use whatever `ADMIN_AUTH` is instead of GoogleAuth.

**Pull request closed without merging**